alvr/server/cpp/platform/linux/EncodePipelineNvEnc.cpp

```diff
@@ -144,3 +144,23 @@ void alvr::EncodePipelineNvEnc::PushFrame(uint64_t targetTimestampNs, bool idr)
         throw alvr::AvException("avcodec_send_frame failed:", err);
     }
 }
+
+bool alvr::EncodePipeline::GetEncoded(FramePacket &packet)
+{
+  av_packet_free(&encoder_packet);
+  encoder_packet = av_packet_alloc();
+  int err = avcodec_receive_packet(encoder_ctx, encoder_packet);
+  if (err != 0) {
+    av_packet_free(&encoder_packet);
+    if (err == AVERROR(EAGAIN)) {
+      return false;
+    }
+    throw alvr::AvException("failed to encode", err);
+  }
+  packet.data = encoder_packet->data;
+  packet.size = encoder_packet->size;
+  packet.pts = encoder_packet->pts;
+  packet.flags = encoder_packet->flags;
+  packet.isIDR = NV_ENC_LOCK_BITSTREAM::pictureType == NV_ENC_PIC_TYPE_IDR;
```

Comment:
`NV_ENC_LOCK_BITSTREAM::pictureType == NV_ENC_PIC_TYPE_IDR` was meant for Windows NVENC.
Here it works with the FFmpeg flag.